src/context/StoreContext.js

```diff
@@ -87,6 +87,20 @@ export default function StoreContextProvider(props) {
     localSetBibleReference(obj)
   }
 
+  const setNewBibleBook = (bId) => {
```

Comment:
Could you rename `bId` so that it's meaning is mor apparent? From the context I gather it's probably a "Book Id" but even that is till ambiguous. 

Another option here is to add a comment about what `setNewBibleBook` is supposed to do

Reply:
I have added a preceded comment

Reply:
I don't see your comment.